charts/redpanda/values.schema.json

```diff
@@ -321,6 +324,114 @@
               "type": "object"
             }
           }
+        },
+        "tieredStoragePersistentVolume": {
```

Comment:
I just learned that this schema exists.  I'm concerned about the maintenance burden of keeping it in sync.

To help folks doing operator/cloud, we added a schema to Redpanda builds for using in situations just like this, e.g.: https://cloudsmith.io/~redpanda/repos/redpanda/packages/detail/raw/redpanda-configuration-schema/22.3.11/

It's in a swagger-like format rather than json-schema, but you could probably convert it pretty easily, and never have to edit this by hand again (yay!)

Reply:
Thinking a bit further... you could probably also generate some of values.yaml this way.  What you get from "rpk cluster config edit" (inlined docs and all) is all just built from the redpanda config schema -- this could be your path to having those inline docs without having to deal with maintaining them.

Reply:
> To help folks doing operator/cloud, we added a schema to Redpanda builds for using in situations just like this, e.g.: https://cloudsmith.io/~redpanda/repos/redpanda/packages/detail/raw/redpanda-configuration-schema/22.3.11/

That's great! My concerns is you provided schema for particular Redpanda configuration version where helm is about to support multiple Redpanda version in Kubernetes environment, so it needs to:

- drop some properties in older versions
- generate `bootstrap.yaml` config for specific range of Redpanda versions
- set properties at runtime not only as properties defined in `bootstrap.yaml` and `redpanda.yaml`
- support properites that are tight to Kuberentes (statefulset, ingress, DNS, TLS secrets etc.)

> Thinking a bit further... you could probably also generate some of values.yaml this way.

Yes, some of the values yes, but it would require custom tool. For me manual copy and paste few properties is easier. At least this PR didn't trigger in me the need for automation 🙃 

> this could be your path to having those inline docs without having to deal with maintaining them.

True. As new feature in this repo defiantly worth considering.